internal/store/bundle.go

```diff
@@ -68,60 +83,29 @@ func (b *bundleStore) Store(ref reference.Reference, bndle *bundle.Bundle) (refe
 	if err = bndle.WriteFile(path, 0644); err != nil {
 		return id, errors.Wrapf(err, "failed to store bundle %q", ref)
 	}
+	appendRef(b.refsMap, id, ref)
 	return id, nil
 }
 
 func (b *bundleStore) Read(ref reference.Reference) (*bundle.Bundle, error) {
-	path, err := b.storePath(ref)
+	paths, err := b.storePaths(ref)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to read bundle %q", ref)
 	}
 
-	data, err := ioutil.ReadFile(filepath.Join(path, "bundle.json"))
+	bndl, err := b.fetchBundleJSON(filepath.Join(paths[0], "bundle.json"))
```

Comment:
What happens to other paths here? Is it worth it returning all the paths if we only use the first (and I guess it has to be the `id` path?